com.liferay.blade.eclipse.provider/src/com/liferay/blade/eclipse/provider/JavaFileJDT.java

```diff
@@ -168,6 +168,8 @@ public boolean visit(TypeDeclaration node) {
 	public SearchResult findImport(final String importName) {
 		final List<SearchResult> searchResults = new ArrayList<>();
 
+		setFile(_file);
```

Comment:
Here you set the file and below you call reset file.  I'm not sure what is the correct thing to do but whatever it is, it should be the same.